runtime/queries/metricsview_aggregation.go

```diff
@@ -166,7 +164,7 @@ func (q *MetricsViewAggregation) Export(ctx context.Context, rt *runtime.Runtime
 	return nil
 }
 
-func (q *MetricsViewAggregation) rewriteToMetricsViewQuery() (*metricsview.Query, error) {
+func (q *MetricsViewAggregation) rewriteToMetricsViewQuery(export bool) (*metricsview.Query, error) {
```

Comment:
Seemed okay to depend on `q.Export`?
Under what condition can `q.Exporting` be true now. I see it being set only from unit tests.

Reply:
Marked `q.Exporting` as deprecated (you're right, it's only used in unit tests to enable testing export formats – would have been nicer to actually call `Export` from the unit test).